### PR TITLE
style!: rename some `CLI`->`Deploy`, decapitalize acronym names

### PR DESCRIPTION
**Pull request by MingweiSamuel**
BREAKING CHANGE: changes some pub-facing names. Behaviors are the same.

### CODE REVIEW

File: hydro_deploy/hydroflow_plus_deploy/src/runtime.rs
Comment (MingweiSamuel) [inline hydro_deploy/hydroflow_plus_deploy/src/runtime.rs:1]: Should these things still be called `Cli*` if it's not `hydro_cli` anymore?

Comment (shadaj) [reply hydro_deploy/hydroflow_plus_deploy/src/runtime.rs:1]: Ooh yes, we should rename to `Deploy*`

Comment (MingweiSamuel) [reply hydro_deploy/hydroflow_plus_deploy/src/runtime.rs:1]: Ok Ill do that here

**Review by shadaj — APPROVED** [review]
Nice!

**Pull request merged**